### PR TITLE
 ENG-8540: Fixing flaky TestStatisticsSuite

### PR DESCRIPTION
**Pull request by akhanzode**

### CODE REVIEW

File: src/frontend/org/voltdb/client/Distributer.java [hunk 1]
```diff
@@ -741,12 +742,22 @@ public void stopping(Connection c) {
...
+                            @Override
+                            public void run() {
+                                subscribeToNewNode();
+                            }
+                        }, new Random().nextInt(RESUBSCRIPTION_DELAY_MS),
+                                TimeUnit.MILLISECONDS);
+                    } catch (RejectedExecutionException ree)
+                    {
+                        // this is for race if m_ex shuts down in the middle of schedule
+                        ree.printStackTrace(); // for testing
```
Comment (manjujames) [inline src/frontend/org/voltdb/client/Distributer.java]: The comment makes it look like this is temporary?

**Comment by manjujames** [comment]
This is just breaking up TestStatisticsSuite to multiple test classes using one base class? It looks good to me. It might be good to use a separate statistics package or something underneath regressionsuites. Could be a first step towards breaking up the tests to meaningful packages..

File: src/frontend/org/voltdb/client/Distributer.java (hunk 1 above)
Comment (akhanzode) [inline src/frontend/org/voltdb/client/Distributer.java]: You are right I will remove this.

**Comment by akhanzode** [comment]
squash merged.

**Pull request closed without merging**